### PR TITLE
Fix pidgin media indictor issue

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Turns out, on test, pidgins media content is under a `video` key, not a `media` key (WAT!)

This change protects us from that by ensuring the media object just exist in a MAP to render a media indicator

A future issue will fix them for pidgin

**Code changes:**

- Dont render media indictor if media key is missing
- Protect against sliceing null if format is missing

**testing:**
using [this data](https://gist.github.com/dr3/07df599215558ca53f8c9ebcba0c4ddd) where media blocks have been swapped with video blocks, it shouldnt error. It wont show media indicators but will fail gracefully, showing a normal promo

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/StoryPromo/MediaIndicator.test.jsx [hunk 1]
```diff
@@ -50,8 +77,18 @@ describe('Story Promo Media Indicator', () => {
     <MediaIndicator item={noDurationItem} />,
   );
 
+  shouldShallowMatchSnapshot(
```
Comment (ghost) [inline src/app/containers/StoryPromo/MediaIndicator.test.jsx:80]: Should this be a `shallowMatch` or a null check?

Comment (dr3) [reply src/app/containers/StoryPromo/MediaIndicator.test.jsx:80]: shallowMatch, as we do render a media indicator if the format is missing, just with no offscreen text :)

**Review by ghost — APPROVED** [review]
Looks great

**Comment by david-boydell** [comment]
This looks good, rfm.

**Comment by david-boydell** [comment]
Cool, I no longer see `Cannot read property 'slice' of null` and the page renders as expected. This is ready for merge.

**Pull request merged**